.github/utils/update_docs.sh

```diff
@@ -10,7 +10,7 @@ invoke create-api-reference-docs --pre-clean
 
 echo -e "\n-o- Commit update - API Reference -o-"
 git add docs/api_reference
-if [ -n "$(git status --porcelain --untracked-files=no --ignored=no docs/api_reference)" ]; then
+if [ -n "$(git status --porcelain --ignored=no docs/api_reference)" ]; then
```

Comment:
@ml-evs you were the one to add these options - do you remember why, exactly?
Since were already targeting the `docs/api_reference` folder here, I don't see why we need to turn these options "off"?

Reply:
It looks like I added the `docs/api_reference` target in https://github.com/Materials-Consortia/optimade-python-tools/commit/f5ee374f945f7d043553e9ca2310f1ab7379e71c and you added the untracked files in https://github.com/Materials-Consortia/optimade-python-tools/pull/496?

Reply:
My bad then. I think the `ignored` option should be removed as well then?

Reply:
Ah, I see now. There was no target in #496, but that would solve the same issue I was "fixing" with that PR. Actually, the target it MUCH better.